client/src/components/navbar/navbar.jsx

```diff
@@ -76,8 +97,18 @@ export default function Navbar() {
       >
         <Toolbar className={classes.toolbar}>
           <Typography variant="h6" noWrap className={classes.toolbarTitle}>
-            <Link href="/" className={classes.link}>
-              Baking Company
+            <Link href="/" className={classes.logolink} style={{}}>
+              <img
+                src={
+                  process.env.PUBLIC_URL +
+                  "/Assets/images/birthday-cake-solid.svg"
+                }
+                width="25"
+                height="25"
+                alt ='logo'
+                className = {classes.logoImage}
+              ></img>
```

Comment:
Any empty tag can be closed like this <img src='' /> instead of adding a close </img> tag.